docs/kb/ttl-facts.rst

```diff
@@ -63,15 +63,15 @@ Facts About Expiring Data
    .. code-block:: none
 
       gc grace seconds: 0
-      sstable A: token range: [10, 10], timestamp range: [1, 1], ttl: 2, max deletion time: 3 (1+2)
-      sstable B: token range: [10, 10], timestamp range: [1, 5], ttl: 2, max deletion time: 7 (5+2)
-      now: 6
+      sstable A: token range: [10, 10], timestamp range: [1:00, 1:00], ttl: 2, max deletion time: 3:00 (1+2)
```

Comment:
The previous code was correct, timestamps as well as deletion time is stored as simple integer on disk.